packages/components/checkboxes/checkboxes.js

```diff
@@ -1,73 +1,114 @@
 const { toggleConditionalInput } = require('../../common')
 
 /**
+ * Checkboxes component
+ *
  * Conditionally show content when a checkbox button is checked
  * Test at http://localhost:3000/nhsuk-frontend/components/checkboxes/conditional.html
  */
-const syncAllConditionalReveals = function syncAllConditionalReveals(input) {
-  const allInputsInForm = input.form.querySelectorAll('input[type="checkbox"]')
-  allInputsInForm.forEach((item) =>
-    toggleConditionalInput(item, 'nhsuk-checkboxes__conditional--hidden')
-  )
-}
+class Checkboxes {
+  constructor($module) {
+    if (!$module) {
+      return this
+    }
 
-/**
- * Uncheck other checkboxes
- *
- * Find any other checkbox inputs with the checkbox group value, and uncheck them.
- * This is useful for when a “None of these" checkbox is checked.
- */
-const unCheckAllInputsExcept = function unCheckAllInputsExcept(input) {
-  const allInputsInSameExclusiveGroup = input.form.querySelectorAll(
-    `input[type="checkbox"][data-checkbox-exclusive-group="${input.getAttribute('data-checkbox-exclusive-group')}"]`
-  )
-
-  allInputsInSameExclusiveGroup.forEach((inputWithSameName) => {
-    const hasSameFormOwner = input.form === inputWithSameName.form
-    if (hasSameFormOwner && inputWithSameName !== input) {
-      inputWithSameName.checked = false // eslint-disable-line no-param-reassign
+    this.$module = $module
+
+    const $inputs = this.$module.querySelectorAll(
+      '.nhsuk-checkboxes .nhsuk-checkboxes__input'
```

Comment:
is the .nhsuk-checkboxes redundant here? I thought $module was the .nhsuk-checkboxes element.